pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerHealthCheck.java

```diff
@@ -50,7 +55,7 @@
       @ApiResponse(code = 503, message = "Broker is not healthy")
   })
   public String getBrokerHealth() {
-    ServiceStatus.Status status = ServiceStatus.getServiceStatus();
+    ServiceStatus.Status status = ServiceStatus.getServiceStatus(_instanceId);
```

Comment:
why did we decide to add checking only the broker health check back?

Reply:
The reason is current ServiceStatus is a map from instance_id to health check callback.
Broker healthcheck should only give the healthcheck of broker, not other component.

Reply:
So doing this will healthcheck just the broker, but I think the broker will still return 200s while it's draining connections.

Reply:
yes, so the healthcheck will be only the broker's lifecycle. It's not linked to PinotServiceManager.

When you send the kill signal:
1. PinotSM will set itself to isShuttingDown=true, so PinotSM healthcheck will Fail. Meanwhile broker healthcheck is still ok. 
2. Then PinotSM will stop Broker, then broker will stop and healthcheck will fail.

So from users' perspective, there is no glitch.